gitops/README.md

```diff
@@ -52,3 +52,69 @@ The PR should have:
 
 - kubeconfig/s of the OpenShift clusters to be placed under gitops/credentials/kubeconfig/compute
 - overlay/kustomization.yaml to be created under `gitops/environment/compute/<clustername>/overlay/kustomization.yaml`
+
+
+## Next steps
+
+Once you have completed the above process, you now have access to a kcp workspace synced with a workload cluster where you can start creating PipelineRuns.
+Furthermore, if you wish to create new workspaces (eg: dev, qe, sre etc), you can create an APIBinding on that workspace to bind to the same workload cluster that you have available. This is possible as kcp creates a APIExport Custom Resource in the workspace where the workload cluster was synced. This mechanism of using APIExport and APIBindings makes it possible to use the same compute (workload cluster) with multiple workspaces.
+
+Below snippet shows the default APIExport CR named 'kubernetes' created in a workspace named 'compute'. 
+
+```
+$ KUBECONFIG=kcpinstance.kubeconfig k ws
+Current workspace is "root:rh-sso-${ORG_ID}:compute".
+
+#Below apiexport CR named 'kubernetes' is created automatically once we sync the workload cluster to the workspace 
+$ KUBECONFIG=kcpinstance.kubeconfig k get apiexports
+NAME         AGE
+kubernetes   46h
+ 
+$ KUBECONFIG=kcpinstance.kubeconfig k get apiexports/kubernetes -o yaml | k neat
```

Comment:
I have not seen us referencing the "kubectl-neat" plugin anywhere. I would just remove its use here.

Reply:
Done.